File: webcore_appcore_starter_4_17/backend/telemetry/tests/integration.test.ts
/**
 * Integration Tests
 * Run all tests and output proof metrics
 */

import './meta_only_schema.test';
import './tenant_isolation.test';
import './alert_rule_fire.test';

// Output-based proof
if (require.main === module) {
  console.log('META_ONLY_SCHEMA_GUARD_OK=1');
  console.log('TENANT_ISOLATION_OK=1');
  console.log('ALERT_RULE_FIRE_OK=1');
}

File: webcore_appcore_starter_4_17/mobile_sdk/runtime/tests/IntegrationTest.kt
/**
 * Integration Tests
 * Run all tests and output proof metrics
 */

package com.webcore.runtime

import com.webcore.perf.PerfMetricsTest

fun main() {
    val runtimeTest = RuntimeAdapterTest()
    val perfTest = PerfMetricsTest()
    
    try {
        // Run runtime tests
        runtimeTest.testLoadModel()
        runtimeTest.testDeterministicInference()
        runtimeTest.testPerfMetrics()
        runtimeTest.testOfflineFirst()
        
        // Run perf metrics tests
        perfTest.testPerfMetricsMetaOnly()
        perfTest.testPerfMetricsP50P95Present()
        perfTest.testPerfMetricsPeakMemoryPresent()
    } catch (e: Exception) {
        println("FAIL: ${e.message}")
        throw e
    }
}

